### DISCUSSION TITLE
Building a train/evaluation pipeline with Bounding Boxes

### DISCUSSION THREAD

**Discussion by jonieva** [Project-MONAI/MONAI#7053] [Q&A] [answered]
Hi!

I'm new to MONAI and I'm trying to train a model that predicts bounding boxes (like RetinaNet, etc.). 

I noticed there are a bunch of bounding box transforms implemented [here](https://github.com/Project-MONAI/MONAI/blob/f214b27264f7a62e0e5ddb3ff131af794b6d484b/monai/apps/detection/transforms/array.py), but I'm a bit confused on how to use them. 

Could anybody point me to an example of how to apply transforms to images+bounding boxes during training, and the corresponding inverted transforms during inference?

Thank you in advance.

**Comment by KumoLiu** [comment, marked as answer]
Hi @jonieva, there is a [folder](https://github.com/Project-MONAI/tutorials/tree/main/detection) for detection in the tutorial repo you could refer to.
Thanks!
